pkg/config/types.go

```diff
@@ -22,15 +22,18 @@ func parseWebhooks(in string) ([]WebhookConfig, error) {
 	return webhookCfgs, nil
 }
 
-func parseStrings(in string) []string {
-	var out []string
+func parseExclusions(in string) (exclusions, inclusions []string) {
 	for _, in := range strings.Split(in, ",") {
 		in := strings.TrimSpace(in)
 		if in != "" {
-			out = append(out, in)
+			if strings.HasPrefix(in, "!") {
```

Comment:
What does inclusions/exclusions look like in the configmap configurations? May want to add it to the PR description.